web/pandas/pdeps/0010-required-pyarrow-dependency.md

```diff
@@ -0,0 +1,67 @@
+# PDEP-10: PyArrow as a required dependency
+
+- Created: 17 April 2023
+- Status: Under discussion
+- Discussion: [#52711](https://github.com/pandas-dev/pandas/pull/52711)
+              [#52509](https://github.com/pandas-dev/pandas/issues/52509)
+- Author: [Matthew Roeschke](https://github.com/mroeschke)
+- Revision: 1
+
+## Abstract
+
+This PDEP proposes that:
+
+- PyArrow becomes a runtime dependency starting pandas 2.1
+- The minimum version of PyArrow supported starting pandas 2.1 is version 7.
+- The minimum version of PyArrow will be bumped every major pandas release to the highest
+  PyArrow version that has been released for at least 2 years, and the minimum PyArrow version will be
+  maintained for every minor version in the major version series.
+
+## Background
+
+PyArrow is an optional dependency of pandas that provides a wide range of supplemental features to pandas:
+
+- Since pandas 0.21.0, PyArrow provided I/O reading functionality for Parquet
+- Since pandas 1.2.0, pandas integrated PyArrow into the `ExtensionArray` interface to provide an optional string data type backed by PyArrow
+- Since pandas 1.4.0, PyArrow provided I/0 reading functionality for CSV
+- Since pandas 1.5.0, pandas provided an `ArrowExtensionArray` and `ArrowDtype` to support all PyArrow data types within the `ExtensionArray` interface
+- Since pandas 2.0.0, All I/O readers have the option to return PyArrow-backed data types, and many methods now utilize PyArrow compute functions to
```

Comment:
```suggestion
- Since pandas 2.0.0, all I/O readers have the option to return PyArrow-backed data types, and many methods now utilize PyArrow compute functions to
```